scss/navLink.module.scss

```diff
@@ -9,3 +12,8 @@
 .noUnderline {
   text-decoration: none !important;
 }
+
+.active {
```

Comment:
Some bootstrap components don't have the bsPrefix or the bsClass so i'm not sure if it's fixable.

Reply:
What proposals did you try out to try to fix the problem of `using !important`? I'm wanting to know what you tried that failed, so that I know not to suggest the methods

Reply:
Alright, i imported the AppNav styles and used them instead because they have higher specificity.